src/_guides/language/analysis-options.md

```diff
@@ -121,15 +127,10 @@ analyzer:
 
 You can use the flags together or separately;
 both default to `true`.
-The presence of either flag, regardless of value, enables
-the Dart 2 type system.
 
-{% comment %}
-**PENDING:
-Will these flags still appear under strong-mode in Dart 2.0?
+**[PENDING: Do these flags still appear in Dart 2.0/2.1?
```

Comment:
If by "these flags" you mean 'implicit-casts' and 'implicit-dynamic', then yes.

Possible future: Leaf has a proposal to split those up into three flags, but that hasn't been decided on.